Makefile

```diff
@@ -347,6 +347,9 @@ set-build-envs:
  	endif
   ifeq ($(DECKHOUSE_REGISTRY_HOST),)
  		export DECKHOUSE_REGISTRY_HOST=registry.deckhouse.io
+  endif
+  ifeq ($(OBSERVABILITY_SOURCE_REPO),)
+  	export OBSERVABILITY_SOURCE_REPO=https://example.com
```

Comment:
Error: template: werfConfig:64:45: executing "werfConfig" at <env "OBSERVABILITY_SOURCE_REPO">: error calling env: the environment variable "OBSERVABILITY_SOURCE_REPO" must be set or default must be provided